templates/base/navbar.html

```diff
@@ -1,5 +1,7 @@
 <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
   <a class="navbar-brand" href="/">Ketabdan Online Publishing</a>
+  <a class="nav-item nav-link active" href="{% url 'shortstories:index' %}">Short Stories</a>
```

Comment:
I was hoping you would figure this out by checking bootstrap example in the link I sent you, it seems the correct format to add additional link is
```
      <li class="nav-item">
        <a class="nav-link" href="#">Link</a>
      </li>
```
please check their example again and see how they were doing it, [link)(http://getbootstrap.com/docs/4.1/components/navbar/#supported-content)

Reply:
You probably need to add this whole block of code, please check the example from the bootstrap documentation

```
<div class="collapse navbar-collapse" id="navbarSupportedContent">
    <ul class="navbar-nav mr-auto">
      <li class="nav-item active">
        <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="#">Link</a>
      </li>```

Reply:
ok, thanks